Formula/ffmpeg.rb

```diff
@@ -119,6 +121,13 @@ def install
     # Needs corefoundation, coremedia, corevideo
     args << "--enable-videotoolbox" if OS.mac?
 
+    # Replace hardcoded default VMAF model path
+    %w[doc/filters.texi libavfilter/vf_libvmaf.c].each do |f|
+      inreplace f, "/usr/local/share/model", HOMEBREW_PREFIX/"share/libvmaf/model"
+      # Since libvmaf v2.0.0, `.pkl` model files have been deprecated in favor of `.json` model files.
+      inreplace f, "vmaf_v0.6.1.pkl", "vmaf_v0.6.1.json"
```

Comment:
This doesn't seem to be fixed on FFmpeg master. Has this been reported upstream?

Reply:
I will consult with the ffmpeg developers on whether they intend to make this configurable.
My understanding is that it is not practical to autodetect the model path, so this is left to packagers to resolve.

Reply:
The related issues have been reported upstream but there appears not to be any related patches in the past year.
https://trac.ffmpeg.org/ticket/9094
https://trac.ffmpeg.org/ticket/9047
https://github.com/Netflix/vmaf/issues/753
https://github.com/Netflix/vmaf/issues/324

Other packagers have opted to replace the path at build time. I asked in `#ffmpeg-devel` and and they agreed that replacing the patch in the Homebrew formula is a reasonable solution for the time being.